public/js/controllers/app_controller.js

```diff
@@ -79,12 +76,8 @@ class Controller {
         DataController.watchATCredits(project, displayATCredits);
     }
 
-    static displaySystems() {
+    static displayServerMetrics() {
```

Comment:
If you're using Miranda in the UI can you call this displayMirandaMetrics instead?

Reply:
updated, thanks.